zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -30,7 +30,6 @@ public class InterpreterOption {
   public static final transient String SHARED = "shared";
   public static final transient String SCOPED = "scoped";
   public static final transient String ISOLATED = "isolated";
-  private static ZeppelinConfiguration conf =  ZeppelinConfiguration.create();
```

Comment:
how about converting this static field from `static` to a member field? then we don't need to change the signature of `getOwners` method.

some thoughts: current change is `inconsistent`, only part of `getXXX` methods require `conf` parameter

Reply:
I don't really like the implementation. See https://github.com/apache/zeppelin/pull/4726/commits/ec3c312682f959107880968148b2080eb6b60803
What do you think?